test/es-module/test-dynamic-import-script-lifetime.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+// Flags: --expose-gc --experimental-vm-modules
```

Comment:
nit: I think keeping the flags on the first line helps notice them, I find it easier to miss when it's "hidden" with other comments.
```suggestion
// Flags: --expose-gc --experimental-vm-modules

'use strict';

```